src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,71 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String model;
+    private String color;
+    private int countDoors;
+    private int countWheels;
+    private Engine engine;
 
+    protected String getModel() {
+        return model;
+    }
+
+    protected void setModel(String model) {
+        this.model = model;
+    }
```

Comment:
why are all getters and setters protected?

Reply:
done